cli/config/configfile/file.go

```diff
@@ -190,15 +191,26 @@ func (configFile *ConfigFile) Save() error {
 	if err != nil {
 		return err
 	}
+	defer func() {
+		temp.Close()
+		if retErr != nil {
+			if err := os.Remove(temp.Name()); err != nil {
+				logrus.WithError(err).WithField("file", temp.Name()).Debug("Error cleaning up temp file")
```

Comment:
Given that in this case, we have exited early, and didn't update the actual `~/.docker/config.json`, perhaps we should print this as a warning?

Something like

```
WARNING: Failed to update config file: error cleaning up temp file 
```

WDYT?

Reply:
We could, but we should already get an error returned.